linode/manager
# fix: [M3-7760] - Create SSH key drawer hidden by Rebuild Linode dialog

## Description 📝
- Quick fix to prevent the SSH Key drawer from being hidden by the Rebuild Linode dialog. We'll potentially look into redesigning the Rebuild Linode / Add a disk from Image flow (in a different ticket) as well

## Target release date 🗓️
9/2

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![image](https://github.com/user-attachments/assets/2547a7a5-8de2-49e5-8461-26f11aedbd77) | ![image](https://github.com/user-attachments/assets/a03151d8-bfe9-459e-957b-65bd35775d42) |

## How to test 🧪

### Reproduction steps
- navigate to a Linode's page, click on three dot menu and Rebuild Linode option. Click 'Add an SSH Key' - drawer is hidden behind dialog

### Verification steps
- confirm drawer no longer hidden behind dialog
- Confirm UI flow for places that use the CreateSSHKeyDrawer still work as expected
  - Create Linode
  - Add a disk to Linode via image
  - My profile >> SSH Keys >> Add an SSH Key

## As an Author I have considered 🤔

*Check all that apply*
